tests/unit/test_stored_procedure.py

```diff
@@ -33,6 +33,7 @@ def test_stored_procedure_execute_as(execute_as):
     fake_session.sproc = StoredProcedureRegistration(fake_session)
     fake_session._plan_builder = SnowflakePlanBuilder(fake_session)
     fake_session._analyzer = Analyzer(fake_session)
+    fake_session._runtime_version = None
```

Comment:
If left undefined, _runtime_version variable will not be found (it is required when setting the RUNTIME_VERSION parameter for registering sprocs/udfs)